notify/impl_test.go

```diff
@@ -10,12 +10,12 @@ import (
 	"github.com/stretchr/testify/require"
 	"golang.org/x/net/context"
 
-	"github.com/prometheus/common/model"
-	"github.com/prometheus/alertmanager/types"
 	"github.com/prometheus/alertmanager/config"
 	"github.com/prometheus/alertmanager/template"
-	"net/url"
+	"github.com/prometheus/alertmanager/types"
+	"github.com/prometheus/common/model"
 	"io/ioutil"
+	"net/url"
```

Comment:
Both `"io/ioutil"` and `"net/url"` should be in the first block of imports in this list, with the other stdlib imports.